manifests/init.pp

```diff
@@ -15,7 +15,7 @@
   $logfile    = undef,
 ) {
   require homebrew
-  $servicename = 'dev.dnsmasq'
+  $servicename = "${tld}.dnsmasq"
```

Comment:
A few of boxen's tools and commands assume that all of its managed services are in the `dev.` namespace. I'd suggest not changing that, even if it ends up serving DNS for a different TLD.